### ISSUE TITLE
[Bug]: SimpleFin-synced account shows wrong total amount. (possibly related to `startDate`

### ISSUE THREAD

**Issue by jeffjose** [actualbudget/actual#3048]
### Verified issue does not already exist?

- [X] I have searched and found no existing issue
- [X] I will be providing steps how to reproduce the bug (in most cases this will also mean uploading a demo budget file)

### What happened?

Looks like the total amount displayed on the top of the page for an account is different is the account is old vs an account created in a fresh budget file.

I believe this is due to starting balances and `startDate`. In the new budget file, the `startDate` doesnt go as back as the original account's starting balance, and that somehow causes the total amount to be different.

In other words
**BUDGET 1 created few months back - ACCOUNT A (simpleFin synced)**
- First transaction date / Starting balance `2023-11-XX`.
- Total balance -$2000 (incorrect)

**BUDGET 2 freshly created - ACCOUNT A (simpleFin synced)**
- First transaction date / Starting balance`2024-04-XX`
- Total balance -$6000 (correct)

### Where are you hosting Actual?

Docker

### What browsers are you seeing the problem on?

Chrome

### Operating System

Linux

**Comment by duplaja** [comment]
Did you set a starting balance for the account, for a particular date? 

The total amount shown at the top is calculated using the sum of all transactions. If you set a starting balance, then the bank sync uses that as the initial date, and goes from there (or 90 days, whichever is most recent).

**Comment by jeffjose** [comment]
No. I didnt set Starting Balance. Both accounts were 100% populated by SimpleFin.

![image](https://github.com/user-attachments/assets/4411c4cd-142f-44c7-8c1d-64123e0394e3)
Image showing starting balance of the "old" account. Note how it starts on 11/13/2023

Same account (credit card) but on a new budget file and synced via SimpleFin today. The starting balance is on 4/15/2025

![image](https://github.com/user-attachments/assets/196ecc52-86ac-4f8a-94a6-24ca7a9ab2f2)

FWIW, the "old" account shows the same transactions on 4/15/2025
![image](https://github.com/user-attachments/assets/8eb7e685-8e0e-47c2-a532-0af60db96e4c)

**Comment by jeffjose** [comment]
Exported transactions into a sheet. I plan to dig into this a bit more later today as well.

https://docs.google.com/spreadsheets/d/1t1Ub28aeJGT909Q8Z1kQrwv80CZF-h9V0oQTwLfTLHE/edit?usp=sharing

**Comment by xshalan** [comment]
I have this issue too. Did you able to fix it?

**Comment by jshargrave** [comment]
I'm also seeing this same issue

**Comment by matt-fidd** [comment]
I can't reproduce this nowadays. My advice to anyone who comes across this in the future is to correct the starting balance transaction if it's incorrect - it's a one time fix and the balance should then be correct going forwards.

**Comment by dnwk** [comment]
I still have this issue. I was having the correct balance before. But for some reason maybe some transaction didn't get synced and balance just go out of sync. There should have an option to just use synced balance.
